esmvalcore/preprocessor/_io.py

```diff
@@ -163,6 +163,18 @@ def _load_zarr(
     ignore_warnings: list[dict[str, Any]] | None = None,
     backend_kwargs: dict[str, Any] | None = None,
 ) -> CubeList:
+    # note on ``chunks`` kwarg to ``xr.open_dataset()``
+    # docs.xarray.dev/en/stable/generated/xarray.open_dataset.html
+    # this is very important because with ``chunks=None`` (default)
+    # data will be realized as Numpy arrays and transferred in memory
+    # both for remote and local Zarr stores;
```

Comment:
```suggestion
    # data will be realized as Numpy arrays and transferred in memory;
```

This is true for all data types (not only Zarr), so I suggest removing this bit